website/docs/Pipelines/ATAC/count-matrix-overview.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 2
+---
+
+# ATAC Count Matrix Overview
+
+The ATAC pipeline's default count matrix output is a h5ad file generated using [SnapATAC2](https://github.com/kaizhang/SnapATAC2) and [AnnData](https://anndata.readthedocs.io/en/latest/index.html). 
+
+The h5ad file contains unstructured metadata (`h5ad.uns`; [Table 1](#table-1-global-attributes)) as well as per-barcode quality metrics (`h5ad.obs`; [Table 2](#table-2-cell-metrics)). It also contains an equivalent gene expression barcode for each ATAC barcode. Raw fragments are stored in the `h5ad.obsm['insertion']` property of the h5ad file.
```

Comment:
At the end here, I would link out to snapatac2's documentation on the import_data function, which was used to generate the h5ad: https://kzhang.org/SnapATAC2/api/_autosummary/snapatac2.pp.import_data.html#snapatac2.pp.import_data

Reply:
Good idea. Thanks @ekiernan!